### PR TITLE
add run apps of currently focussed vm and dmenu background coloring

### PR DESCRIPTION
**Pull request by jampe**
- implement dmenu_path vm command allowing you to run apps of the currently focussed vm
- implement dmenu background coloring according to the vm it's showing the menu for

### CODE REVIEW

**Comment by raffaeleflorio** [comment]
Hi @jampe ,
Thank you for this contribution! The patch seems fine.
As last thing, can you sign your commits? Here you are a useful guide: https://www.qubes-os.org/doc/code-signing/

File: dmenu.patch [hunk 1]
```diff
@@ -185,66 +172,97 @@ index 338bac4..0fb0a84
...
++		template_vm_name=`qvm-prefs $1 template`
++		echo $template_vm_name
++	else
++		echo $1
++	fi
++}
++
 +function usage
 +{
 +    echo "usage: $0 dom0 | vms" >&2
```
Comment (raffaeleflorio) [inline dmenu.patch]: @jampe can you update this function to reflect the new `vm` argument?

**Comment by jampe** [comment]
Sure, commit is signed 👍

**Review by raffaeleflorio — CHANGES_REQUESTED** [review]

**Review by raffaeleflorio — APPROVED** [review]

**Comment by raffaeleflorio** [comment]
Merged! 😄

**Pull request merged**